### DISCUSSION TITLE
error in running scenario_runner

### DISCUSSION THREAD

**Discussion by nadrainam** [carla-simulator/carla#4295] [Q&A]
Hi
I try to run the basic scenario by using the command
python3 scenario_runner.py --scenario FollowLeadingVehicle_1 --reloadWorld 0.0
but I got the following error
![scenario_runnererror](https://user-images.githubusercontent.com/79971063/121519072-8d4f0700-c9f1-11eb-8518-6d9e7f8d69d1.png)

mean while I also run the manual_control.py

Please anyone can help what is the problem.

Thanks in advance

**Comment by glopezdiest** [comment]
Hey @nadrainam. Could you please tell me the versions you are using fpr CARLA, ScenarioRunner and py_trees? I can't say for sure but this seems like a missmatch of py_trees version (version 0.8.3 is required).

**Reply by nadrainam** [reply]
Hi
The versions are as follows:
Carla 0.911 (as i can see it in the folder :/pyhtonAPI/Carla/Dist)
py_trees version 2.1.4 ( check it with the command pip3 show py-trees| greb Version)
Scenario_runner 0.9.11 ( I check it through command: python3 scenario_runner --help)

I try to install the py-trees version 0.8.3 by using command : pip3 install py-trees=0.8.3
But now when I check the version it is giving me 2.1.4.

I do not know how to install py-trees version 0.8.3

Thank you so much for your reply
